src/language-service/src/home-assistant/haConnection.ts

```diff
@@ -117,7 +117,7 @@ export class HaConnection implements IHaConnection {
     }
   };
 
-  private getHassEntities = async (): Promise<HassEntities> => {
+  public getHassEntities = async (shouldLog = true): Promise<HassEntities> => {
```

Comment:
Revert that when a simple axios GET call outside language service is implemented.